### ISSUE TITLE
GMAIL SMTP is not working. (Verified)

### ISSUE THREAD

**Issue by PeopleInside** [LiveHelperChat/livehelperchat#943]
Hi @remdex,
Followed the guide
https://livehelperchat.com/how-to-use-google-smtp-in-live-helper-chat-204a.html
but gmail SMTP seems are not working.
Tested with a gmail account who don't have two factor authentication active, pop is active also imap.

Two different user reporting issue with gmail.
Today I have tested and I AM also unable to see gmail work.
Can you verify? 

Tested also by following the App password link but is not active in the account so when access to this page I get error as there are not two step authentication active.

From the forum: https://forum.livehelperchat.com/viewtopic.php?id=2215

Also there are issue on testing email function:
https://github.com/LiveHelperChat/livehelperchat/issues/937

Tried also to look at the old issue:
https://github.com/LiveHelperChat/livehelperchat/issues/558
but nothing also change template from address doesn't seems made gmail work

**Comment by PeopleInside** [comment]
I was able to find an email sent from gmail account however all time I saved settings for gmail I see error showed from LHC so I will keep this open.
Also is a priority fix the test email function as if not works seems email are not working when is.

**Comment by PeopleInside** [comment]
Today I have assisted a member who have asked help for configure SMTP.
I was unable to make gmail works.

Don't know if this is really an issue with gmail but the user said all was solved by using a yahoo email account.
I have seen in remote help that the user has not the two factor authentication activated in the gmail account. Pop and  IMAP was not active but the gmail account was working well on Thunderbird.

Don't know if there are really an issue with gmail accounts.
https://forum.livehelperchat.com/viewtopic.php?pid=9100#p9100
